sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/sqlCompiler/compiler/backend/rust/ToRustVisitor.java

```diff
@@ -864,8 +865,12 @@ public VisitDecision preorder(DBSPConstantOperator operator) {
                 .append(operator.getName())
                 .append(" = ")
                 .append("circuit.add_source(Generator::new(|| ");
+        this.builder.append("if Runtime::worker_index() == 0 {");
```

Comment:
I forgot how the worker ordinals work. Is there a worker-0 on each machine when using scale-out or is there only one worker-0 across all machines?

Reply:
Right now we don't really have scale-out.
Each operator has N workers.
On a single machine N is the same for all operators.
How it will be for multiple machines it's not yet clear.
But 0 should always exist.
One thing I don't understand is whether the output has to be sharded in some way.